test/test-cat.bats

```diff
@@ -0,0 +1,81 @@
+#!./test/libs/bats/bin/bats
+
+load 'libs/bats-support/load'
+load 'libs/bats-assert/load'
+load 'helpers'
+
+setup() {
+  setupNotesEnv
+}
+
+teardown() {
+  teardownNotesEnv
+}
+
+notes="./notes"
+
+@test "Should show created note" {
+  touch "$NOTES_DIRECTORY/note.md"
+  echo line1 >> "$NOTES_DIRECTORY/note.md"
+  echo line2 >> "$NOTES_DIRECTORY/note.md"
+  run $notes cat note.md
+
+  assert_success
+  assert_output $'line1\nline2'
+}
+
+@test "Accepts names without .md to show" {
+  touch "$NOTES_DIRECTORY/note.md"
+  echo line1 >> "$NOTES_DIRECTORY/note.md"
+  echo line2 >> "$NOTES_DIRECTORY/note.md"
+  run $notes cat note
+
+  assert_success
+  assert_output $'line1\nline2'
+}
+
+@test "Should fail to show non-existent note" {
+  run $notes cat note
+
+  assert_failure
+}
+
+@test "Accepts relative notes paths to show" {
+  touch $NOTES_DIRECTORY/note.md
```

Comment:
Here (and I think everywhere in this file?) you don't need the `touch`, since you're echoing straight afterwards it's not actually doing anything.

Reply:
Indeed, good point.
Fixed now